src/bilder/images/edxapp/prebuild.py

```diff
@@ -0,0 +1,31 @@
+from pyinfra.operations import files, git
+
+from bilder.components.baseline.steps import install_baseline_packages
+from bilder.images.edxapp.lib import WEB_NODE_TYPE, node_type
+
+install_baseline_packages(
+    packages=[
+        "curl",
+        "git",
+        "libmariadbclient-dev",
+        "python3-pip",
+        "python3-venv",
+        "python3-dev",
+        "build-essential",
+    ],
+    upgrade_system=True,
+)
+
+if node_type == WEB_NODE_TYPE:
+    files.directory(
+        name="Ensure themes directory is present",
+        path="/edx/app/edxapp/themes/",
+        present=True,
+    )
+    git.repo(
+        name="Load theme repository",
+        src="https://github.com/mitodl/mitxonline-theme",
```

Comment:
Should we also make this generic or be part of the config so that it can be used with other builds?

Reply:
Once we start expanding this to work in other environments we will make that value configurable. For the time being I'm fine with it being hard-coded for the sake of simplicity.